hexrd/ui/image_stack_dialog.py

```diff
@@ -367,11 +368,13 @@ def build_data(self):
                 'max_file_frames': self.state['max_file_frames'],
                 'max_total_frames': self.state['max_total_frames']
             },
-            'reverse_frames': self.state['reverse_frames']
+            'reverse_frames': self.state.get('reverse_frames', False)
         }
-        if not self.state['omega_from_file']:
+        if not self.state['omega_from_file'] and len(self.state["wedges"]):
```

Comment:
```suggestion
        if not self.state['omega_from_file'] and self.state['wedges']:
```

If `self.state['wedges']` will be a list, dict, string, or `None`, then this is the [pep8 recommended](https://www.python.org/dev/peps/pep-0008/) way to check it.

Reply:
Thank you, fixed.